__mocks__/multisigGraphInfo.ts

```diff
@@ -105,6 +105,17 @@ export const multisigEntries2 = multisigAccountGraphInfoDTO2.multisigEntries.map
         ),
 );
 
+export const multisigEntries3 = multisigAccountGraphInfoDTO.multisigEntries.map(
+    (multisigAccountInfoDTO) =>
+        new MultisigAccountInfo(
+            1,
+            multisigAccountInfoDTO.multisig.accountAddress,
+            multisigAccountInfoDTO.multisig.minApproval,
+            multisigAccountInfoDTO.multisig.minRemoval,
+            [],
```

Comment:
just wondering, in `cosignatoryAddresses` it should contain a multisig account, right?
to build multi-level multisig?

Reply:
yes, but having multisig addresses would let the account be a multisig one resulting empty array of children.